appveyor.yml

```diff
@@ -0,0 +1,30 @@
+build: false
+platform:
+  - x64
+
+environment:
+  COMPOSER_NO_INTERACTION: "1"
+  ANSICON: '121x90 (121x90)'
+  matrix:
+    - PHP_VERSION: '7.2'
+      COMPOSER_FLAGS: '--prefer-stable --prefer-lowest'
+    - PHP_VERSION: '7.3'
+    - PHP_VERSION: '7.4'
```

Comment:
PHP version matrix needs to be updated.

Or maybe AppVeyor is not required at this point? Github Actions support Windows 🤔.